File: 1455-check-if-a-word-occurs-as-a-prefix-of-any-word-in-a-sentence/1455-check-if-a-word-occurs-as-a-prefix-of-any-word-in-a-sentence.py
class Solution:
    def isPrefixOfWord(self, sentence: str, searchWord: str) -> int:
        sentence = sentence.split()
        output = -1
        for word in range(len(sentence)):
            if sentence[word][: len(searchWord)] == searchWord:
                output = word+1
                break
        return output

File: 1455-check-if-a-word-occurs-as-a-prefix-of-any-word-in-a-sentence/1455-check-if-a-word-occurs-as-a-prefix-of-any-word-in-a-sentence.java
class Solution {
    public int isPrefixOfWord(String sentence, String searchWord) {
        
        String[] sentences = sentence.split(" ");
	    int searchWordLength = searchWord.length();
	    int output = -1;
		
		for(int i=0;i<sentences.length;i++){
		    if (sentences[i].length() >= searchWordLength) {
		        if(sentences[i].substring(0,searchWordLength).equals(searchWord)) {
		            output = i+1;
		            break;
		        }
		    }
		}
        return output;
    }
}